pcap-to-parquet/src/main/java/nl/sidn/pcap/load/LoaderThread.java

```diff
@@ -184,12 +184,16 @@ private void archiveFile(String pcap) {
         throw new RuntimeException("creating archive dir: " + archiveDir.getAbsolutePath());
       }
     }
-    if (file.renameTo(
-        new File(archiveDir.getPath() + System.getProperty("file.separator") + file.getName()))) {
-      LOGGER.info(file.getName() + " is archived!");
-    } else {
-      throw new RuntimeException("Error moving " + file.getName() + " to the archive");
-    }
+    File newFile = new File(archiveDir.getPath() + System.getProperty("file.separator") + file.getName());
+    try {
+		if (Files.move(Paths.get(file.getAbsolutePath()), Paths.get(newFile.getAbsolutePath())) != null) {
+			LOGGER.info(pcap + " is archived!");
+		} else {
+			throw new RuntimeException("Error moving " + file.getName() + " to the archive");
```

Comment:
I'm not entirely sure if the if else statement is necessary here since I don't know if Files.move can exit without moving the file while also not throwing an error which would be caught by the try catch statement.

Reply:
Looking at documentation and examples I see no situation where Files.move would return null instead of throwing an exception. Removed if else statement.